package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "html-dnd",
-  "version": "1.2.0",
+  "name": "xl-html-dnd",
+  "version": "0.0.5",
```

Comment:
Sorry for my too late reply... :bow:

What purpose of the changes at line 2 and 3 is?

Reply:
it should be reverted, it's not supposed to be for a PR.
I had to release a separate lib to make it work, as my PR wasn't accepted ;)

Reply:
OK, we can take either of the 2 ways:

1. Revert some commits on this PR
2. Create new PR that include squashed commit

Which way do you like?